script/install-skywire.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=install-skywire service
+After=systemd-networkd.service network.target armbian-firstrun.service armbian-resize-filesystem.service systemd-user-sessions.service network-online.target systend-networkd-wait-online.service NetworkManager-wait-online.service
+Before=skymanager.service
+Wants=network-online.target
+
+[Service]
+Type=oneshot
+User=root
+Group=root
+ExecStart=/usr/bin/install-skywire
```

Comment:
Is this a binary or a script? If its a script, I would use an extension.

Reply:
this is not actually used or included in any package or image currently.

in the future if we want to have skywire installed on the first boot of the image, it may be useful.

it is a script but nothing in /usr/bin has an extension. You want to explicitly block it from being run as a regular command, without specifying the full path?

Reply:
removed now